### PR TITLE
Add carthage support by adding json files

### PR DESCRIPTION
**Pull request by carolanitz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,7 +88,25 @@ Then, run the following command,
 ```bash
 $ pod install
 ```
+### Carthage
 
+[Carthage](https://github.com/Carthage/Carthage) is  a way to add frameworks to your Cocoa application. You can install it with the following command,
```
Comment (mkchoi212) [inline README.md]: There are two spaces between `is` and `a way to` :D

Comment (Mikanbu) [reply README.md]: Additionally, I think we should add Carthage to the table of contents at the top.

**Review by fkuehne — APPROVED** [review]
Looks great! (assuming that Carthage supports tar.xz)

**Comment by carolanitz** [comment]
since his was merged yes :D https://github.com/Carthage/Carthage/pull/2548

**Pull request merged**